Simpson_Assign1/Program.cs

```diff
@@ -16,11 +16,131 @@ static void Main(string[] args)
             Students = InitializeStudents();
             Courses = InitializeCourses();
 
+            string input = "";
```

Comment:
You can actually just use "var" for the type anytime you are declaring and assigning a variable in the same line.  This is similar to the "auto" keyword in c++.  It will automatically set the type based on what was assigned to it.

No need to change any of your variable declarations, just a useful FYI.